deploy/metric-adapter/apiservice.yaml

```diff
@@ -24,4 +24,19 @@ spec:
   version: v1beta2
   insecureSkipTLSVerify: true
   groupPriorityMinimum: 100
-  versionPriority: 200
\ No newline at end of file
+  versionPriority: 200
+
+---
+apiVersion: apiregistration.k8s.io/v1
+kind: APIService
+metadata:
+  name: v1beta1.external.metrics.k8s.io
+spec:
+  service:
+    name: metric-adapter
+    namespace: crane-system
+  group: external.metrics.k8s.io
+  version: v1beta1
+  insecureSkipTLSVerify: true
+  groupPriorityMinimum: 100
+  versionPriority: 100
```

Comment:
add ApiService for v1beta2

Reply:
> add ApiService for v1beta2

There is no v1beta2 version for External metrics